pkg/kubelet/dockertools/docker.go

```diff
@@ -534,19 +533,29 @@ var (
 	ErrContainerCannotRun = errors.New("Container cannot run")
 )
 
-func inspectContainer(client DockerInterface, dockerID, containerName, tPath string) (*api.ContainerStatus, error) {
+// Internal information kept for containers from inspection
+type containerStatusResult struct {
+	status api.ContainerStatus
+	podIP  string
```

Comment:
Dawn I understand the overall intent. But in this specific case, we are inspecting a container and that method is returning a pod IP. That seems weird.